drivers/adc/adc_ad559x.c

```diff
@@ -272,24 +286,29 @@ static int adc_ad559x_init(const struct device *dev)
 	return 0;
 }
 
-static const struct adc_driver_api adc_ad559x_api = {
-	.channel_setup = adc_ad559x_channel_setup,
-	.read = adc_ad559x_read,
-#ifdef CONFIG_ADC_ASYNC
-	.read_async = adc_ad559x_read_async,
-#endif
-	.ref_internal = AD559X_ADC_VREF_MV,
-};
+/* clang-format off */
```

Comment:
Why the clang-format disable?
If it is because of the weird formatting after IF_ENABLED, you could try moving this section to the end of the struct.